test/integration/scheduler/scheduler_test.go

```diff
@@ -228,7 +228,7 @@ kind: Policy
 				},
 				"ScorePlugin": {
 					{Name: "NodeResourcesBalancedAllocation", Weight: 1},
-					{Name: "PodTopologySpread", Weight: 1},
+					{Name: "PodTopologySpread", Weight: 2},
```

Comment:
I would not have expected a configuration change here

Reply:
Explained in https://github.com/kubernetes/kubernetes/pull/91598/files?file-filters%5B%5D=.go&file-filters%5B%5D=.json#r433494354